packages/lib/har/harUtils.ts

```diff
@@ -84,22 +87,50 @@ export const toHarContent = (body: string | Buffer | null, mimeType?: string): H
 };
 
 export const fromHarContent = (content?: HarFormatContent) => {
-  if (content?.text) {
+  if (content?.text !== undefined) {
     return Buffer.from(content.text, content.encoding === 'base64' ? 'base64' : 'binary');
   }
+  if (content?.json !== undefined) {
+    return Buffer.from(stringifyPretty(content.json), 'utf8');
+  }
   return Buffer.alloc(0);
 };
 
+export const checkMimeTypeListAndParseBody = (
+  parseMimeTypesAsJson: string[],
+  body: string | Buffer,
+  mimeType?: string,
+): HarFormatPostData => {
+  const defaultTextReturn = {
+    mimeType: mimeType ?? '',
+    text: body.toString('binary'),
```

Comment:
To prevent `body.toString('binary')` from being evaluated and then unused (in case the body is correctly parsed as a json object), you should move this to the end of the function.
To avoid duplicated code (line 118 and 121), you should empty the content of the catch block, so the execution will continue to the end of the function.